README.md

```diff
@@ -56,6 +56,41 @@ $ oc get deploy -nopenshift-compliance
 $ oc get pods -nopenshift-compliance
 ```
 
+### Deploying with Helm
+
+The repository contains a [Helm](https://helm.sh/) chart that deploys the
+compliance-operator. This chart is currently not published to any official
+registries. You're required to run the chart from this repository.
+
+Make sure you create the namespace prior to running `helm install`:
+
+```
+$ kubectl create namespace openshift-compliance
+```
+
+Next, deploy a release of the compliance-operator using `helm install` from
+`deploy/compliance-operator-chart/`:
+
+```
+$ cd deploy/compliance-operator-chart
+$ helm install --namespace openshift-compliance --generate-name .
```

Comment:
This failed for me on the helm version I had installed (probably old):

```$ helm version
Client: &version.Version{SemVer:"v2.14.1", GitCommit:"5270352a09c7e8b6e8c9593002a73535276507c0", GitTreeState:"clean"}
Error: could not find tiller```

Helm 3.7.2 worked, so it might be good to have a note that you need at least Helm v3